### ISSUE TITLE
Skips animation or white screen when using Navigation 2.0 packages.

### ISSUE THREAD

**Issue by nyck33** [NearHuscarl/flutter_login#277]
After implementing go_router in my app, now the animation skips.
The transition is very abrupt.  
https://youtu.be/cPIZNEs9pNQ

**Comment by nyck33** [comment]
I now tried routemaster and I get stuck on a white screen on the first login attempt but all subsequent ones are fine.

**Comment by nyck33** [comment]
I am using AWS Cognito and storing tokens and userdata in SharedPreferences and I checked that they were empty.  Is there any part of your code that needs to be "warmed up" before working correctly?  I saw someplace you are using timers and saying "Well, it works so..."

**Comment by nyck33** [comment]
My best guess is that it's not working well with Navigation 2.0.

**Comment by nyck33** [comment]
In Android, if I go back to Android homescreen and reopen the app, it proceeds to my homescreen normally.  In web, I have to refresh the browser.

**Comment by nyck33** [comment]
@NearHuscarl Can you give an option to turn off some animations?  This package is great but for the time being I want to just simplify it as much as possible.

**Comment by nyck33** [comment]
Does this package have some sort of mechanism to go all the way back to void main() after onSubmitAnimationComplete()?  Due to the animation problem, I wrote my own loginscreen and replaced yours in my app and now it is not going all the way back to void main so I must rewrite some logic that seemed to be specific to your package.  Ie. with my login page, it does not go back to void main but when I push a route, it goes straight to that page (using routemaster).  With your package, I had to save the logged in user to shared preferences on the login screen and have a whole sequence to retrieve the user from shared preferences in void main.
